File: 백준/Bronze/1292. 쉽게 푸는 문제/쉽게 푸는 문제.py
# 수열 만들기
lst = []
for i in range(1000):
    if len(lst) <= 1000:
        for _ in range(i):
            lst.append(i)
    else:
        break

a, b = map(int, input().split())
print(sum(lst[a-1:b]))

File: 백준/Bronze/1292. 쉽게 푸는 문제/쉽게 푸는 문제.js
const fs = require("fs");
const [A, B] = fs.readFileSync("/dev/stdin").toString().split(" ").map(Number);

const array = [];
for (let i = 0; i <= 1000; i++) {
  if (i <= 1000) {
    for (let j = 0; j < i; j++) {
      array.push(i);
    }
  }
}

let sum = 0;
for (let i = A - 1; i < B; i++) {
  sum += array[i];
}

console.log(sum);
